ui/app/templates/vault/cluster/access/leases/list.hbs

```diff
@@ -115,25 +115,19 @@
     </div>
   {{/each}}
 {{else}}
-  <div class="box is-bottomless has-background-white-bis">
-    <div class="columns is-centered">
-      <div class="column is-half has-text-centered">
-        <div class="box is-shadowless has-background-white-bis">
-          <p class="has-text-grey">
-            {{#if (eq baseKey.id '')}}
-              There are currently no leases.
-            {{else}}
-              {{#if filterIsFolder}}
-                {{#if (eq filter baseKey.id)}}
-                  There are no leases under <code>{{filter}}</code>.
-                {{else}}
-                  We couldn't find a prefix matching <code>{{filter}}</code>.
-                {{/if}}
-              {{/if}}
-            {{/if}}
-          </p>
-        </div>
-      </div>
+  <div class="empty-state">
+    <div class="empty-state-title">
+      {{#if (eq baseKey.id '')}}
+        There are currently no leases.
+      {{else}}
+        {{#if filterIsFolder}}
+          {{#if (eq filter baseKey.id)}}
+            There are no leases under <code>{{filter}}</code>.
+          {{else}}
+            We couldn't find a prefix matching <code>{{filter}}</code>.
+          {{/if}}
+        {{/if}}
+      {{/if}}
```

Comment:
Can this one not use `<EmptyState />`? Even maybe

```hbs
{{! ... kind of scribbled in here, but hopefully you can follow what I mean }}
{{#if (eq baseKey.id '')}}
  <EmptyState />
{{ else }}
  <EmptyState />
  {{#if filterIsFolder}}
    <EmptyState />
  {{ else }}
    <EmptyState />
  {{/if}}
  {{! ... etc etc }}
{{/if}}
```
..right now if you change the component in any way, you also have to come and change this also.

Reply:
That was a big question I had. Do we move some of this logic so that the template can use the component or is it better to do this in some cases? I don't want to overoptimize, but I also don't like these non-component versions running around potentially not getting updated later.

Reply:
Yeah, if it was me I'd wouldn't want the non-component version running around amongst component versions (strange images of scurrying components in my head 😂 ). Matthew might have reason to want it different though, he has more context, see what he thinks. Looking at it a bit longer now I'd wonder how much you are gaining by making this a component actually - it's probably the same quandary you are thinking about. But if you are gonna go component, I'd say try and keep it consistent and use it everywhere.

Will leave it with you and Matthew though.

Reply:
Yep, agree with @johncowen on this that we should try to keep everything tucked away in the component. Probably a good opportunity to move the title to a computed property in the controller so that we don't need the nest of ifs - I can do that if you'd like!

Reply:
@meirish Sure! That would be great, thanks for the help. I almost moved it there but didn't want to before checking. That will be much nicer on the template side